### PR TITLE
Fix error in `@fluid-tools/benchmark`'s `CHANGELOG`

### PR DESCRIPTION
**Pull request by jikim-msft**
#### Description

`@fluid-tools/benchmark` starting from `^0.49.0` provides running benchmark tests on custom data. 

Found an error in the `CHANGELOG` and fixed the description to the correct [tag name](https://github.com/microsoft/FluidFramework/blob/16494160f63266e9a89508f534f5d55195ad0d2b/tools/benchmark/src/mocha/customOutputRunner.ts#L40) to avoid confusion.

### CODE REVIEW

File: tools/benchmark/CHANGELOG.md [hunk 1]
```diff
@@ -8,8 +8,7 @@
 
 ## 0.49.0
 
-Provide @benchmarkCustom feature to log custom measurements. To profile custom usage, define tests using the `benchmarkCustom()` function. The argument `run` to the function includes a reporter with `addMeasurement()`
-to write custom data to report.
+Provides @CustomBenchmark feature to log custom measurements. To profile custom usage, define tests using the `benchmarkCustom()` function. The argument `run` to the function includes a reporter with `addMeasurement()` to write custom data to report.
```
Comment (alexvy86) [inline tools/benchmark/CHANGELOG.md:11]:
```suggestion
Adds a feature to run benchmarks that log custom measurements. To use it, define tests using the `benchmarkCustom()` function. The `run` argument passed to the function is itself a function that will get passed a reporter object with an `addMeasurement()` method, that the test code can use to define custom data to report as the benchmark output. These custom-measurement benchmarks get the string `@CustomBenchmark` appended to their title, so they can be run selectively using `mocha`'s `fgrep` flag.
```

**Review by alexvy86 — COMMENTED** [review]
Suggested a rewrite with a bit more detail. The other two files shouldn't be updated in this PR, correct?

File: tools/benchmark/CHANGELOG.md (hunk 1 above)
Comment (Josmithr) [reply tools/benchmark/CHANGELOG.md:11]: Nit: I also always recommend line-breaking along sentence boundaries. Makes future edits/diffs easier to review. :)

File: tools/benchmark/CHANGELOG.md [hunk 2]
```diff
@@ -8,8 +8,7 @@
 
 ## 0.49.0
 
-Provide @benchmarkCustom feature to log custom measurements. To profile custom usage, define tests using the `benchmarkCustom()` function. The argument `run` to the function includes a reporter with `addMeasurement()`
-to write custom data to report.
+Adds a feature to run benchmarks that log custom measurements. To use it, define tests using the `benchmarkCustom()` function. The `run` argument passed to the function is itself a function that will get passed a reporter object with an `addMeasurement()` method, that the test code can use to define custom data to report as the benchmark output. These custom-measurement benchmarks get the string `@CustomBenchmark` appended to their title, so they can be run selectively using `mocha`'s `fgrep` flag.
```
Comment (Josmithr) [inline tools/benchmark/CHANGELOG.md:11]: Nit
```suggestion
Adds a feature to run benchmarks that log custom measurements.
To use it, define tests using the `benchmarkCustom()` function.
The `run` argument passed to the function is itself a function that will get passed a reporter object with an `addMeasurement()` method, that the test code can use to define custom data to report as the benchmark output.
These custom-measurement benchmarks get the string `@CustomBenchmark` appended to their title, so they can be run selectively using `mocha`'s `fgrep` flag.
```

Comment (Josmithr) [inline tools/benchmark/CHANGELOG.md:11]: Nit: might be worth adding an example of the mocha fgrep usage.

**Review by Josmithr — APPROVED** [review]

File: tools/benchmark/CHANGELOG.md [hunk 3]
```diff
@@ -8,8 +8,14 @@
 
 ## 0.49.0
 
-Provide @benchmarkCustom feature to log custom measurements. To profile custom usage, define tests using the `benchmarkCustom()` function. The argument `run` to the function includes a reporter with `addMeasurement()`
-to write custom data to report.
+<<<<<<< HEAD
```
Comment (alexvy86) [inline tools/benchmark/CHANGELOG.md:11]: Merge conflict :)

**Pull request merged**